src/components/ShareDialog.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Typography,
+} from '@material-ui/core';
+import {
+  FacebookShareButton,
+  FacebookIcon,
+  EmailShareButton,
+  EmailIcon,
+  LinkedinShareButton,
+  LinkedinIcon,
+  RedditShareButton,
+  RedditIcon,
+  TwitterShareButton,
+  TwitterIcon,
+} from 'react-share';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    background: theme.palette.background.secondary,
+    color: 'white',
+    textAlign: 'center',
+  },
+  shareIcon: {
+    marginRight: theme.spacing(1),
+  },
+  copyText: {
+    color: '#808080',
+  },
+}));
+
+export default function ShareDialog(props) {
+  const classes = useStyles();
+  const { open, onClose, link, title } = props;
+
+  return (
+    <Dialog open={open} onClose={onClose}>
+      <DialogTitle className={classes.root}>SHARE</DialogTitle>
+      <DialogContent className={classes.root}>
+        <DialogContentText className={classes.root}>
+          Currently Sharing: {title}
```

Comment:
Why does this show up like this:
![image](https://user-images.githubusercontent.com/8023233/104117293-02000700-52d5-11eb-82ac-6275680a8def.png)

Reply:
I believe because of my mistake of including the `https://` in the wrong variable